extensions/HyperVExtension/src/HyperVExtension/Providers/HyperVProvider.cs

```diff
@@ -126,6 +126,10 @@ public ComputeSystemAdaptiveCardResult CreateAdaptiveCardSessionForDeveloperId(I
         try
         {
             _hyperVManager.StartVirtualMachineManagementService();
+            var imageList = _vmGalleryService.GetGalleryImagesAsync().GetAwaiter().GetResult();
+            var virtualMachineHost = _hyperVManager.GetVirtualMachineHost();
+
+            return new ComputeSystemAdaptiveCardResult(new VMGalleryCreationAdaptiveCardSession(imageList, _stringResource, virtualMachineHost));

```

Comment:
Consider a new variable as it would be better to read and debug.

Reply:
sure  I can do that